indicator-doom-cpu

```diff
@@ -15,15 +16,23 @@ You should have received a copy of the GNU General Public License along
 with this program.  If not, see <http://www.gnu.org/licenses/>.
 '''
 APP_NAME = 'DOOM CPU Indicator'
-APP_AUTHOR = 'Edward G. Bruck <ed.bruck1@gmail.com>'
 APP_WEBSITE = 'https://github.com/ebruck/indicator-doom-cpu'
 APP_VERSION = '1.0.0'
+APP_COPYRIGHT = '\n'.join([c for c in str.splitlines(APP_LICENCE) if c.startswith("Copyright")])
+APP_AUTHORS = [a[10:] for a in str.splitlines(APP_LICENCE) if a.startswith("Copyright")]
+
+print(APP_AUTHORS)
```

Comment:
May want to remove this print statement.

Reply:
oops, missed that one from my debugging.  One moment...